package.json

```diff
@@ -36,58 +46,74 @@
   },
   "homepage": "https://github.com/fastify/fastify-cli#readme",
   "engines": {
-    "node": ">= 10"
-  },
-  "standard": {
-    "ignore": [
-      "test/data/parsing-error.js",
-      "test/data/undefinedVariable.js"
-    ]
+    "node": ">= 12"
   },
   "dependencies": {
-    "chalk": "^4.1.2",
-    "chokidar": "^3.5.2",
-    "close-with-grace": "^1.1.0",
-    "commist": "^2.0.0",
-    "dotenv": "^16.0.0",
+    "@fastify/autoload": "^5.0.0",
+    "@fastify/sensible": "^5.0.0",
+    "@oclif/core": "^1.8.2",
+    "@oclif/plugin-help": "^5.1.12",
+    "ansi-colors": "^4.1.3",
+    "chokidar": "^3.5.3",
+    "ejs": "^3.1.8",
     "fastify": "^4.0.0-rc.2",
-    "generify": "^4.0.0",
-    "help-me": "^3.0.0",
-    "is-docker": "^2.0.0",
-    "make-promises-safe": "^5.1.0",
-    "pino-colada": "^2.2.2",
-    "pkg-up": "^3.1.0",
-    "pump": "^3.0.0",
+    "fastify-plugin": "^3.0.0",
+    "inquirer": "^8.2.4",
+    "is-docker": "^3.0.0",
+    "pkg-up": "^4.0.0",
     "resolve-from": "^5.0.0",
-    "semver": "^7.3.5",
-    "split2": "^4.1.0",
-    "yargs-parser": "^20.0.0"
+    "semver": "^7.3.7",
+    "tslib": "^2.4.0"
   },
   "devDependencies": {
-    "@fastify/autoload": "^5.0.0",
-    "@fastify/sensible": "^4.1.0",
-    "@types/node": "^17.0.8",
-    "@types/tap": "^15.0.5",
-    "concurrently": "^7.0.0",
-    "del-cli": "^3.0.1",
-    "fastify-plugin": "^3.0.0",
-    "fastify-tsconfig": "^1.0.1",
-    "minimatch": "^5.1.0",
-    "pre-commit": "^1.2.2",
-    "proxyquire": "^2.1.3",
-    "rimraf": "^3.0.2",
-    "simple-get": "^4.0.0",
-    "sinon": "^14.0.0",
-    "standard": "^17.0.0",
-    "strip-ansi": "^6.0.1",
-    "tap": "^16.1.0",
-    "ts-node": "^10.4.0",
-    "ts-standard": "^11.0.0",
-    "tsd": "^0.16.0",
-    "typescript": "^4.5.4",
-    "walker": "^1.0.8"
+    "@types/ejs": "^3.1.1",
+    "@types/inquirer": "^8.2.1",
+    "@types/node": "^17.0.0",
+    "@types/semver": "^7.3.9",
+    "@types/tap": "^15.0.0",
+    "@typescript-eslint/eslint-plugin": "^4.0.1",
+    "eslint": "^7.12.1",
+    "eslint-config-standard-with-typescript": "^21.0.1",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^5.0.0",
+    "oclif": "^3.0.1",
+    "pino-pretty": "^7.6.1",
+    "shx": "^0.3.4",
+    "tap": "^16.2.0",
+    "ts-node": "^10.7.0",
+    "typescript": "~4.4.0"
```

Comment:
this is pretty old

Reply:
Fixed in `~4.4.0` just because of `eslint` and `ts-standard`.

Reply:
ts-standard seems to be barely maintained, are we sure we don't want to rely on something more actively supported? being locked into old TS is really not great

Reply:
> ts-standard seems to be barely maintained, are we sure we don't want to rely on something more actively supported? 

In here, I would like to share the same style with other packages. I am aware of `ts-standard` is leak of maintaince, but it does the job right now.
Currently, I am just waiting for https://github.com/standard/eslint-config-standard-with-typescript update.

In my projects, I bumped the `typescript` version without problem. I only gives me warning of unsupported `typescript`.
As a tools that will be widely used, I don't think this warning is acceptable and I will wait for the update in `standard` first.